packages/ui/src/validators/modals/validatorCard/ValidatorDetail.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { ButtonPrimary } from '@/common/components/buttons'
+import { MarkdownPreview } from '@/common/components/MarkdownPreview'
+import { ModalFooter } from '@/common/components/Modal'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { SidePaneBody, SidePaneLabel, SidePaneRow, SidePaneText } from '@/common/components/SidePane'
+import { NumericValueStat, StatisticsThreeColumns } from '@/common/components/statistics'
+import { TextSmall } from '@/common/components/typography'
+
+import { Validator } from '../../types'
+
+interface Props {
+  validator: Validator
+}
+
+export const ValidatorDetail = ({ validator }: Props) => {
+  return (
+    <>
+      <SidePaneBody>
+        <Details gap={24}>
+          <RowGapBlock gap={4}>
+            <h6>Key elements</h6>
+            <ModalStatistics>
+              <Stat size="s" value={20000}>
+                <TextSmall lighter>Total reward</TextSmall>
+              </Stat>
+              <Stat size="s" value={validator.isVerified ? 'Verified' : 'Unverified'}>
+                <TextSmall lighter>Status</TextSmall>
+              </Stat>
+              <Stat size="s" value={validator.slashed.toString()}>
+                <TextSmall lighter>Slashed</TextSmall>
+              </Stat>
+              <Stat size="s" value={0}>
+                <TextSmall lighter>Uptime</TextSmall>
+              </Stat>
+              <Stat size="s" value={validator.APR.toString()}>
+                <TextSmall lighter>Average APR</TextSmall>
+              </Stat>
+              <Stat size="s" value={validator.staking.others.length}>
+                <TextSmall lighter>Nominators</TextSmall>
```

Comment:
In the design this is  "Staked by nominators":
![image](https://github.com/Joystream/pioneer/assets/6571453/d9b65464-c14c-4e8b-a710-cb966779484f)

And this should also show as an amount of JOY